### PR TITLE
added parachain selection modal

### LINKED ISSUE #247: Input field breaks if a letter is added
To break the input field → enter letters, afterwards no more chars (even numbers) can be input. Only option is to reload the form.

### PR DESCRIPTION
**Pull request by Bullrich**
Added a modal where the user can select a default chain, a predefined chain or add their own.

image

This closes #247

### CODE REVIEW

File: client/src/lib/utils/networkData.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	getChainName(id: number): string | null;
+}
+
+export const Rococo: NetworkData = {
+	networkName: "Rococo",
+	chains: [
+		{ name: "Relay Chain", id: -1 },
+		{ name: "One", id: 1 },
+		{ name: "Two", id: 2 },
+		{ name: "Three", id: 3000 }
```
Comment (Bullrich) [inline client/src/lib/utils/networkData.ts:18]: @paritytech/opstooling which chains should we add here?

Comment (rzadp) [reply client/src/lib/utils/networkData.ts:18]: I don't know how to get a list of all possible ones, but perhaps scrolling through the faucet matrix chats and seeing which parachains people use would be a good trail?

File: .github/workflows/tag-client-pr.yml
Comment (Bullrich) [inline .github/workflows/tag-client-pr.yml:1]: @paritytech/ci I just added the env var because it was missing for the execution

**Review by Bullrich — COMMENTED** [review]
PR has been finished. It is ready for review.

For the network value, the relay chain uses `-1` but when doing the request, it checks if it's `< 0`, if so, it just doesn't send the custom chain id.

**Review by rzadp — APPROVED** [review]

**Pull request merged**